diff --git a/Godbert/Settings.cs b/Godbert/Settings.cs
index ad7f946..543183d 100644
--- a/Godbert/Settings.cs
+++ b/Godbert/Settings.cs
@@ -12,7 +12,7 @@ public class Settings {
         public static Settings Default { get; } = Load();
 
         private static string FileName => Path.Combine(
-            Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Godbert", "settings.json");
+            AppDomain.CurrentDomain.BaseDirectory, "AppData", "settings.json");
 
         private static Settings Load() {
             try {
